cypress/integration/portal-dashboard/portal-dashboard-feedback.spec.js

```diff
@@ -255,6 +255,42 @@ context("Portal Dashboard Feedback Panel", () => {
       });
     });
   });
+  context('API Validation', () => {
+    beforeEach(() => {
+      cy.visit("/?portal-dashboard");
+      cy.get('[data-cy=navigation-select]').click();
+      cy.get('[data-cy="list-item-feedback-report"]').should('be.visible').click();
+      // Mock Firestore
+      cy.window().then((win) => {
+        win.firebase = {
+          firestore: () => ({
+            doc: () => ({
+              set: cy.stub().resolves()
+            })
+          })
+        };
+      });
+    });
+
+    it('displays "No feedback yet" when a student has no feedback', () => {
```

Comment:
"No feedback" doesn't match the text string this test checks for. Change it to "This student hasn't started yet."? Or maybe make the description more generic?